earn/src/pages/boost/ManageBoostPage.tsx

```diff
@@ -0,0 +1,76 @@
+import { useContext, useEffect } from 'react';
+
+import { ethers } from 'ethers';
+import { useParams } from 'react-router-dom';
+import { boostNftAbi } from 'shared/lib/abis/BoostNFT';
+import AppPage from 'shared/lib/components/common/AppPage';
+import { Text } from 'shared/lib/components/common/Typography';
+import { ALOE_II_BOOST_NFT_ADDRESS } from 'shared/lib/data/constants/ChainSpecific';
+import { useChainDependentState } from 'shared/lib/data/hooks/UseChainDependentState';
+import { Address, useContractRead, useProvider } from 'wagmi';
+
+import { ChainContext } from '../../App';
+import BoostCard from '../../components/boost/BoostCard';
+import { sqrtRatioToTick } from '../../data/BalanceSheet';
+import { BoostCardInfo, BoostCardType, fetchBoostBorrower } from '../../data/Uniboost';
+
+const DEFAULT_COLOR0 = 'white';
+const DEFAULT_COLOR1 = 'white';
+
+export default function ManageBoostPage() {
+  const { activeChain } = useContext(ChainContext);
+  const { nftTokenId } = useParams();
+  const provider = useProvider({ chainId: activeChain.id });
+  const [cardInfo, setCardInfo] = useChainDependentState<BoostCardInfo | null>(null, activeChain.id);
+
+  const numericNftTokenId = ethers.BigNumber.from(nftTokenId || 0);
+
+  const { data: boostNftAttributes } = useContractRead({
+    abi: boostNftAbi,
+    address: ALOE_II_BOOST_NFT_ADDRESS[activeChain.id],
+    functionName: 'attributesOf',
+    args: [numericNftTokenId],
+    enabled: !!numericNftTokenId,
+  });
```

Comment:
Usually we use "numeric" to mean Javascript numbers, not `BigNumber`s. I'd suggest the following:

```suggestion
  const { data: boostNftAttributes } = useContractRead({
    abi: boostNftAbi,
    address: ALOE_II_BOOST_NFT_ADDRESS[activeChain.id],
    functionName: 'attributesOf',
    args: [ethers.BigNumber.from(nftTokenId || 0)],
    enabled: nftTokenId !== undefined,
  });
```